nipype/interfaces/io.py

```diff
@@ -440,27 +440,20 @@ def _check_s3_base_dir(self):
             is not a valid S3 path, defaults to '<N/A>'
         """
 
-        # Init variables
-        s3_str = "s3://"
+        # NOTE: due to pathlib.Path in traits.Directory, S3 bucket paths
+        #       follow the format 's3:/bucket_name/...'
+        s3_str = "s3:/"
         bucket_name = "<N/A>"
         base_directory = self.inputs.base_directory
 
         if not isdefined(base_directory):
             s3_flag = False
             return s3_flag, bucket_name
 
-        # Explicitly lower-case the "s3"
+        # Check for 's3:/' in base dir
```

Comment:
how about using: `s3_flag = base_directory.startswith('s3://')` makes it more explicit and potentially avoids other prefixes `ds://` would also say `s3_flag` is true.

Reply:
I don't think other prefixes would cause problems but cleaned the code a bit with your suggestion.